docs/_posts/2018-10-01-how-chinese-learn-english.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "中国人应该怎么学英语？"
+author: yong-chen
+categories: news
+tags:
+image:
```

Comment:
The image is very important. We need to find a good picture to make the post sexier.